### PR TITLE
Nui fixes

### PR DESCRIPTION
**Pull request by Youwotma**

### CODE REVIEW

File: portiaui/app/components/inspector-panel.js [hunk 1]
```diff
@@ -45,6 +45,26 @@ export function getAttributeList(element) {
     return attributeList;
 }
 
+export function getDefaultAttribute(element) {
+    // TODO: Remove attributes that have been already annotated
```
Comment (ruairif) [inline portiaui/app/components/inspector-panel.js]: Attributes can be annotated multiple times. Would you be removing them just to make selecting the default attribute more useful?

File: portiaui/app/services/dispatcher.js [hunk 2]
```diff
@@ -205,14 +205,11 @@ export default Ember.Service.extend({
         });
         if (element) {
             annotation.addElement(element);
+            attribute = attribute || getDefaultAttribute(element);
```
Comment (michalmo) [inline portiaui/app/services/dispatcher.js]: can you make it set the annotation type to url and image for these cases too?

File: portiaui/app/components/inspector-panel.js (hunk 1 above)
Comment (Youwotma) [inline portiaui/app/components/inspector-panel.js]: True, not sure what the most common use case would be, if annotating the same attribute twice, or annotating different attributes. There are few cases where you want different attributes (href and content, src and title) but also few use cases for annotating the same attribute twicd.

At the moment when an element is already annotated the default action with the magic tool enabled will be to select the existing annotation anyway

**Comment by michalmo** [comment]
needs discussion on spider renaming

**Pull request merged**